### ISSUE TITLE
OK to include this in https://lokl.dev ?

### ISSUE THREAD

**Issue by leonstafford** [grrr-amsterdam/simply-static-deploy#37]
Hey Grrr's, 

Awesome work on this. Would you mind if I include it in [https://lokl.dev's](https://lokl.dev) base images?

Trying to bundle all the WP SSGs and companion tools in there.

(sorry to raise issue, no Discussions on repo)

Cheers,

Leon

**Comment by HammenWS** [comment]
@leonstafford Thank you :) First I was a bit skeptical because I read 'Loki', the God of Mischief.

But of course you may include it. But I would recommend the latest version that we just released.
I am curious how you experience the latest release.

**Comment by leonstafford** [comment]
Haha! Awesome, @HammenWS, thanks!

I've got a bit of tidy up on some of my other projects before I do next bundling of default plugins into mischievous Lokl, but will test it out then. I've got the beginnings of a nice end to end testing framework using ShellSpec and the Docker images, so I'll aim to get some automated tests for main export flows with your SimplyStatic.

Only issue I could see come up would be on any namespacing/function conflicts with say regular SimplyStatic if that goes in, too. I've got a SimplerStatic which if I update it may still be worth including, but also seeing Patrick has taken over Simply Static repo here: https://staticword.press/t/simply-static-being-updated and I don't really need any extra projects to work on :D

If Lokl is handy for any of your team or users, just let me know what features you want. Probably not great for agency dev workflows, as no filesystem syncing between container and host, but technically possible with tmux/vim currently included and plans to add in a web based code editor *eventually*. 

Will close this to get out of your way and can ping your GH user from within any of my projects, like when I push out the new release with it included.

Cheers!
